frontend/src/features/admin-form/create/payment/PaymentDrawer.tsx

```diff
@@ -51,6 +62,8 @@ const formatCurrency = new Intl.NumberFormat('en-SG', {
   maximumFractionDigits: 2,
 }).format
 
+const DESCRIPTION_TOOLTIP = 'Description will be reflected on payment receipt'
```

Comment:
Why not just follow the terminology on @staceytan1998's design here? Also for the label itself (commented below)
```suggestion
const DESCRIPTION_TOOLTIP = 'Name will be reflected on payment receipt'
```

Reply:
The cognitive load of understanding that the payment "Name" understood by users is the "description" understood by Stripe's API should be borne by us if this is better for users.

Reply:
Primarily because currently in our backend we still refer to it as 'description' I think the change from description to 'name' should be an issue by itself, and should be reflected throughout our system

Reply:
Oh, my point is that we can keep the "description" terminology internally since its consistent with what stripe API accepts, just change what the user sees on frontend so they can better understand what it means. So no need a separate issue.

Reply:
My other concern is if eventually we do end up having a 'description' or 'sub-description' (currently it was removed by Amit) field as seen in Stacey's previous design. Then we might end up with further confusion.

Reply:
Upon thinking through it, I think I agree, will make the modifications and add comments to highlight the change in definitions from backend schema to frontend display. Thanks @LinHuiqing

Reply:
[aa9d346](https://github.com/opengovsg/FormSG/pull/5998/commits/aa9d3465942aec055d626177146e41c6b4aed2d2)